### PR TITLE
About Command

### PR DESCRIPTION
**Pull request by SolarysDev**
Self explanatory.
The user can determine whether to show bot or server information with the flags `-g[uild]` and `-b[ot]`.

From experience this should work, but please test it anyway, since I literally cant, reason below.
reason
LocalDB won't run on Linux, mssql implementations on Arch are essentially broken.
#139 👀👀👀👀👀👀👀

Implements #126.

### CODE REVIEW

File: src/Volvox.Helios.Core/Modules/Command/Commands/AboutCommand.cs [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                case null:
+                    embed.WithTitle("So you want to hear my life story, ey?")
+                        .AddField(".NET Core Runtime Version", Environment.Version)
+                        .AddField("Bloat Factor", $"{GC.GetTotalMemory(false) / 1048576}MB", true)
+                        .AddField("Server Count", context.Client.Guilds.Count, true);
+                    await context.Channel.SendMessageAsync("", false, embed.Build());
+                    break;
+                default:
+                    embed.WithTitle("So you want to hear my life story, ey?")
+                        .AddField(".NET Core Runtime Version", Environment.Version)
```
Comment (BillChirico) [inline src/Volvox.Helios.Core/Modules/Command/Commands/AboutCommand.cs]: Can we use assembly version here instead? See Bot.cs

File: src/Volvox.Helios.Core/Modules/Command/Commands/AboutCommand.cs [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                        .WithImageUrl(channel.Guild.IconUrl)
+                        .AddField("Owner", $"{channel.Guild.Owner.Username} ({channel.Guild.Owner.Mention})")
+                        .AddField("User Count", channel.Guild.MemberCount)
+                        .AddField("Voice Region", channel.Guild.VoiceRegionId, true);
+                    break;
+                case "-b":
+                case "-bot":
+                case null:
+                    embed.WithTitle("So you want to hear my life story, ey?")
+                        .AddField(".NET Core Runtime Version", Environment.Version)
```
Comment (BillChirico) [inline src/Volvox.Helios.Core/Modules/Command/Commands/AboutCommand.cs]: Can we use assembly version here instead? See Bot.cs

File: src/Volvox.Helios.Core/Modules/Command/Commands/AboutCommand.cs (hunk 1 above)
Comment (SolarysDev) [reply src/Volvox.Helios.Core/Modules/Command/Commands/AboutCommand.cs]: fixed

File: src/Volvox.Helios.Core/Modules/Command/Commands/AboutCommand.cs (hunk 2 above)
Comment (SolarysDev) [reply src/Volvox.Helios.Core/Modules/Command/Commands/AboutCommand.cs]: fixed

**Pull request merged**